scss/_variables.scss

```diff
@@ -1505,15 +1505,15 @@ $navbar-badge-margin-top:                   .375rem !default;
 // End mod
 
 // scss-docs-start navbar-dark-variables
-$navbar-dark-border-color:          $gray-700 !default;
-$navbar-dark-color:                 $white !default;
-$navbar-dark-hover-color:           $primary !default;
-$navbar-dark-active-color:          $primary !default;
-$navbar-dark-disabled-color:        $gray-400 !default;
-$navbar-dark-toggler-border-color:  transparent !default;
-// Boosted mod: np $navbar-dark-icon-color
+$navbar-dark-border-color:          $gray-700 !default; // Boosted mod
+$navbar-dark-color:                 $white !default; // Boosted mod: instead of `rgba($white, .55)`
+$navbar-dark-hover-color:           $primary !default; // Boosted mod: instead of `rgba($white, .75)`
+$navbar-dark-active-color:          $primary !default; // Boosted mod: instead of `$white`
+$navbar-dark-disabled-color:        $gray-700 !default; // Boosted mod: instead of `rgba($white, .25)`
```

Comment:
Is this change voluntary even for the navbars we already had before this PR?

![Screenshot 2023-08-30 at 09 01 23](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/assets/17381666/12270e8e-9ef0-497e-8490-3efbcfe22a94)

Reply:
Since we don't have any spec on this I took the liberty to use the common disabled color we have (the actual on main is #ccc). I was done on purpose for all navbars including local-nav, footer, GH, supra. Should I mention it in the migration guide ?

Reply:
Just had the answer of designers: 
> Yes you're right, it should be #666666.